### DISCUSSION TITLE
Welcome to ndit-website Discussions!

### DISCUSSION THREAD

**Discussion by royemosby** [nditcommunity/ndit-website#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to plan and build our web presence. We hope that you:
  * Read and abide by our [code of conduct](https://github.com/nditcommunity/ndit-code-of-conduct)
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by hayleyw7** [comment]
Great idea. I just linked to this space in the official [README](https://github.com/nditcommunity/ndit-website).
